src/org/folio/rest/Deployment.groovy

```diff
@@ -64,6 +64,11 @@ class Deployment extends GeneralParameters {
         okapi.cleanupServicesRegistration()
     }
 
+    void test() {
+        okapi.configureLdpDbSettings(tenant, admin_user, "asfsdfa")
+        okapi.configureLdpSavedQueryRepo(tenant, admin_user, "asjkfasdf")
```

Comment:
What is 3rd argument mean?

Reply:
It was added for testing purpose, already dropped